account/account_test.go

```diff
@@ -723,21 +655,22 @@ func TestTransactionHashDeclare(t *testing.T) {
 		ExpectedErr  error
 	}
 	testSet := map[string][]testSetType{
-		"mock": {{
-			// Note this is a testnet / goerli transaction
-			Txn: rpc.DeclareTxnV2{
-				Nonce:             utils.TestHexToFelt(t, "0xb"),
-				MaxFee:            utils.TestHexToFelt(t, "0x50c8f3053db"),
-				Type:              rpc.TransactionType_Declare,
-				Version:           rpc.TransactionV2,
-				Signature:         []*felt.Felt{},
-				SenderAddress:     utils.TestHexToFelt(t, "0x36437dffa1b0bf630f04690a3b302adbabb942deb488ea430660c895ff25acf"),
-				CompiledClassHash: utils.TestHexToFelt(t, "0x615a5260d3d47d79fba87898da95cb5394b181c7d5097bc8ced4ed06ac24ac5"),
-				ClassHash:         utils.TestHexToFelt(t, "0x639cdc0c42c8c4d3d805e56294fa0e6bf5a584ad0fcd538b843cc294913b982"),
+		"mock": {
+			{
+				// Note this is a testnet / sepolia transaction
```

Comment:
This is a goerli txn

Reply:
Didn't find this line in the code. Hence, considered fixed.